jetstream/external_config.py

```diff
@@ -54,9 +54,10 @@ def from_github_repo(cls) -> "ExternalConfigCollection":
 
         for file in files:
             if file.name.endswith(".toml"):
+                commits = repo.get_commits(file.path)
                 slug = os.path.splitext(file.name)[0]
                 spec = AnalysisSpec.from_dict(toml.loads(file.decoded_content.decode("utf-8")))
-                last_modified = parser.parse(str(file.last_modified))
+                last_modified = parser.parse(str(commits[0].commit.author.date))
```

Comment:
Fixes https://github.com/mozilla/jetstream/issues/266

Reply:
Once we have ~50 external config files, this will start throwing exceptions since the Github API only allows 50 requests per hour. I guess once we start seeing these issues we might need to switch to authenticated API requests